modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java

```diff
@@ -51,6 +53,29 @@ public static GridClient startClient(GridClientConfiguration clientCfg) throws E
         return client;
     }
 
+    /**
+     * @return Cluster information to show user for.
+     */
+    default String getFullClusterInfo(GridClientClusterState clientCfg){
```

Comment:
This method doesn't actually return full cluster info. Method name should be changed